### DISCUSSION TITLE
CRUD Events

### DISCUSSION THREAD

**Discussion by jellycow** [filamentphp/filament#107] [Help] [answered]
Hi,

just been thinking about a use case where, upon creation, update or deletion of resource(s)/record(s), the changes need to be synced or sent to a third party api for instance. I can't see any information on lifecycle event hooks in the documentation and was wondering if there was already a way to do this and, if not, would it be a useful idea?

**Comment by ryangjchandler** [comment, marked as answer]
@jellycow - could you not use Laravel's model events? That's exactly what they're designed for.

**Reply by jellycow** [reply]
Yup, that was my first though as well but was just pondering on the use of a higher level abstraction - but you are right, it would be pretty pointless & a tad over engineered :-)
